api/src/controllers/record-utils.js

```diff
@@ -79,21 +65,19 @@ const addError = (record, error) => {
  *
  * return unix timestamp integer or undefined
  */
-const parseSentTimestamp = str => {
```

Comment:
this function was way more complicated than it needed to be, taking a number and turning it into a string to then regex detect it _was_ a number to convert it into a date to convert it back into the same number it was in the first place.